python/test/test_polars.py

```diff
@@ -42,6 +42,13 @@ def example_lf(margin=None, **kwargs):
     return lf_domain, lf
 
 
+def test_expr_domain():
+    lf_domain, _ = example_lf()
+    dp.expr_domain(lf_domain, context="filter")
+    with pytest.raises(dp.OpenDPException):
+        dp.expr_domain(lf_domain, context="X")
```

Comment:
Slight preference to split this into a separate test: More informative failure messages, and coverage even if the first test fails.

Having a typo in the context value seems like an easy mistake to make: Hopefully the error message is useful to the end user? If it is, consider making that part of the `pytest.raises` check... but if it's not useful to the end user right now, file an issue, and and TODO referencing the issue here?